frontend/packages/metal3-plugin/src/components/dashboard/events-card.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { FirehoseResource, FirehoseResult } from '@console/internal/components/utils';
+import { EventModel, MachineModel, NodeModel } from '@console/internal/models';
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '@console/internal/components/dashboard/dashboard-card';
+import { EventsBody } from '@console/internal/components/dashboard/events-card/events-body';
+import {
+  EventKind,
+  K8sResourceKind,
+  MachineKind,
+  referenceForModel,
+  K8sKind,
+} from '@console/internal/module/k8s';
+import { DashboardItemProps } from '@console/internal/components/dashboards-page/with-dashboard-resources';
+import { getName, getNamespace, getMachineNodeName } from '@console/shared';
+
+import { getHostMachineName } from '../../selectors';
+import { BaremetalHostModel } from '../../models';
+
+const eventsResource: FirehoseResource = { isList: true, kind: EventModel.kind, prop: 'events' };
+const getMachineResource = (name: string, namespace: string): FirehoseResource => ({
+  isList: false,
+  namespace,
+  name,
+  kind: referenceForModel(MachineModel),
+  prop: 'machine',
+});
+
+const machesInvolvedObject = (
+  kind: K8sKind,
+  name: string,
+  namespace: string,
+  event: EventKind,
+): boolean =>
+  name &&
+  _.isMatch(event.involvedObject, {
+    kind: referenceForModel(kind),
```

Comment:
This does not seem to work because event.involvedObject is 'model.kind' value rather than result of `referenceForModel(kind)`.

these changes made it to work:
```
const machesInvolvedObject = (
  kind: string,
  name: string,
  namespace: string,
  event: EventKind,
): boolean =>
  name &&
  _.isMatch(event.involvedObject, {
    kind,
    name,
    namespace,
  });

// and calling it:
machesInvolvedObject(BaremetalHostModel.kind, getName(host), getNamespace(host), event) || ...
```

Reply:
ok, done :)